src/openrct2/core/Console.cpp

```diff
@@ -19,6 +19,7 @@ extern "C"
     #include "../platform/platform.h"
 }
 
+#include <android/log.h>
```

Comment:
Having them go to android logs is nice in case we need to debug something. Looks like that already is the case, so why can't it stay as is?